spec/controllers/hyrax/dashboard/collections_controller_spec.rb

```diff
@@ -147,14 +184,7 @@
   end
 
   describe "#update" do
-    let(:listener) { Hyrax::Specs::SpyListener.new }
```

Comment:
`listener` is used in multiple places so it was moved outside of the `#update` tests.